Modules/Projects/Core/Strings_TownEvent.json

```diff
@@ -12,7 +12,7 @@
 		{
 			"Key": "EVENT_NAKED_ARRESTED",
 			"Text": [
-				"He hands his halberd to his partner, then confidently marches over to you and snatches you by the scruff of your neck. The {guard} forcefully pushes and pins you against the smooth and cold surface of the nearest wall, then pulls back his visor. An assertive and authoritative, yet also predatory, pair of eyes meets your gaze. \"Hold still, citizen. In order to preserve public order, I am forced to... temporarily remove your presence until you come to your senses.\""
+				"{^guard.subject} hands {guard.possessive} halberd to {guard.possessive} partner, then confidently marches over to you and snatches you by the scruff of your neck. The {guard} forcefully pushes and pins you against the smooth and cold surface of the nearest wall, then pulls back {guard.possessive} visor. An assertive and authoritative, yet also predatory, pair of eyes meets your gaze. \"Hold still, citizen. In order to preserve public order, I am forced to... temporarily remove your presence until you come to your senses.\""
```

Comment:
This works, so you can leave it as-is, but for the record, the game always capitalizes the first character of a log message, so the `^` isn't strictly necessary. See `GameUI.cs:206`.